Pull_Request_and_Code_Review_Standards.md

```diff
@@ -0,0 +1,46 @@
+# Pull Request and Code Review Standards
+
+-   Unless it's a trivial change (spelling mistake, code formatting, etc.), there should be an existing issue that will be closed by the given PR.
+
+-   The branch name from which PR is being made should follow the convention: \`\<issue_number\>-short*-*description\`.
+
+-   The title for the PR should provide a brief description about the changes.
+
+-   The PR should be linked to the issue(s) that it targets, either by using keywords (like `fixes`, `closes`, etc.) in the PR description or by using \`development\` feature.
```

Comment:
Here we could add a link to the GitHub help topic for this:
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

Reply:
Good idea! Done.